app/core/models.py

```diff
@@ -352,3 +352,15 @@ class InterestList(TimeStampedModel):
                                          blank=True)
     name = models.CharField(max_length=200,
                             help_text="Enter the name of the list")
+
+
+class SavedFilter(TimeStampedModel):
+    """Model for Saved Filter"""
+
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
No pk / unique id?

Reply:
All covered, Django creates an PK for us in this case.